### DISCUSSION TITLE
How to add a Serilog Sink, start to finish?

### DISCUSSION THREAD

**Discussion by lboening** [TomasBouda/PoShLog#22] [Show and tell]
I couldn't figure out how you have added a Serilog sink in PowerShell dialect. Could you provide a run-down on how you do it?

**Comment by TomasBouda** [comment]
Here is wiki page I put together, where I tryed to describe how to create Serilog sink in PowerShell - [https://github.com/PoShLog/PoShLog/wiki/Creating-Sink](https://github.com/PoShLog/PoShLog/wiki/Creating-Sink)
I'll keep updating it, if you have any question or feedback, feel free to reply here.
Good luck 👍
